flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentITCase.scala

```diff
@@ -294,6 +294,27 @@ class TableEnvironmentITCase(tableEnvName: String) {
     assertFirstValues(sinkPath)
   }
 
+  @Test
+  def testExecuteExplainSelect(): Unit = {
```

Comment:
Please refer to `TableEnvironmentITCase#testSqlUpdateAndToDataStream`. Godfrey has already offer a better way to check the result.